pwem/wizards/wizard.py

```diff
@@ -560,7 +560,7 @@ def _createControls(self, frame):
         downButton.grid(row=1, column=1, padx=5, pady=5, sticky='ew')
 
     def getDownsample(self):
-        return float(self.downVar.get())
+        return 1
```

Comment:
I do not see the big picture here.....but I'd say this code doesn't seem right?
we have a self.downVar (String) and old code seemed reasonable, and not returning 1 always?